packages/boot/test/bootstrapTests/net-ibc-upgrade.test.ts

```diff
@@ -26,42 +27,52 @@ export const makeTestContext = async t => {
     { cacheSourceMaps: false },
     s => import(s),
   );
-  const swingsetTestKit = await makeSwingsetTestKit(t.log, bundleDir, {
-    configSpecifier: PLATFORM_CONFIG,
+
+  const { handleBridgeSend } = makeMockBridgeKit({
+    ibcKit: { handleOutboundMessage: () => String(undefined) },
+  });
+
+  const swingsetTestKit = await makeCosmicSwingsetTestKit({
+    configSpecifier: '@agoric/vm-config/decentral-itest-vaults-config.json',
+    fixupConfig: config => ({
+      ...config,
+      bundleCachePath: bundleDir,
+    }),
+    handleBridgeSend,
   });
```

Comment:
I'm seeing `ibcKit: { handleOutboundMessage: () => String(undefined) }` repeated a lot; is it necessary to avoid runtime errors? If it's only for avoiding type errors, then let's just make sure to mark `ibcKit` as optional in `makeMockBridgeKit`, but otherwise, we might as well provide it as an overridable default to simplify call sites.
```suggestion
  const swingsetTestKit = await makeCosmicSwingsetTestKit({
    configSpecifier: '@agoric/vm-config/decentral-itest-vaults-config.json',
    fixupConfig: config => ({
      ...config,
      bundleCachePath: bundleDir,
    }),
  });
```
and if necessary:
```patch
 export const makeMockBridgeKit = ({
-  ibcKit,
+  ibcKit = { handleOutboundMessage: () => 'undefined' },
   outboundMessages = new Map(),
   storageKit = makeFakeStorageKit(''),
 }: Partial<MockBridgeOptions> = {}) => {
```